### PR TITLE
tuxpaint, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Loading saved images doesn't seem to work, and I can't check if printing works myself (VirtualBox)

File: media-gfx/tuxpaint/tuxpaint-0.9.23.recipe [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+DESCRIPTION="Tux Paint is a free, award-winning drawing program for children \
+ages 3 to 12 (for example, preschool and K-6). Tux Paint is used \
+in schools and around the world as a computer literacy drawing activity. \
+It combines an easy-to-use interface, fun sound effects, and an \
+encouraging cartoon mascot who guides children as they use the program."
+HOMEPAGE="http://www.tuxpaint.org/"
+COPYRIGHT="2002-2018 Bill Kendrick and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/tuxpaint/files/tuxpaint/$portVersion/tuxpaint-$portVersion.tar.gz"
```
Comment (fbrosson) [inline media-gfx/tuxpaint/tuxpaint-0.9.23.recipe]: Suggestion:
```diff
-SOURCE_URI="http://sourceforge.net/projects/tuxpaint/files/tuxpaint/$portVersion/tuxpaint-$portVersion.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/tuxpaint/tuxpaint-$portVersion.tar.gz"
```

**Comment by scottmc** [comment]
open a new issue for the saved images issue so it doesn't get forgotten, but let's merge this one now.

**Comment by Begasus** [comment]
New issue committed :) https://github.com/haikuports/haikuports/issues/3045

**Pull request merged**